src/components/Combobox/Static/Combobox.tsx

```diff
@@ -127,10 +132,11 @@ const ComboboxInner = <T extends Option, V extends Option | string>(
             textOverflow="ellipsis"
             title={isString(value) ? value : value?.label}
             {...props}
-            {...getInputProps({
-              id,
-              ref,
-            })}
+            {...inputProps}
+            // There is mismatch between desert box onChange type and downshift on change event
+            onChange={
+              inputProps.onChange as unknown as FormEventHandler<HTMLElement>
```

Comment:
ok so if this is repeated among combobox (Dynamic & Static) so how about creating a function that transforms desert box change into downshift change event? sort of adapter?